frontend/templates/views/partials/survey.j2

```diff
@@ -1,11 +1,14 @@
 {% if survey_questions %}
+  [% set surveys = {{ survey_questions|tojson }} %]
+  [% set fixedQuestions = {{fixed_questions|tojson }} %]
+  [% set url = "{{ doc.url }}" %]
```

Comment:
Alternatively you could also do

```suggestion
  [% set url = requestPath %]
```

This is given to the SSR context here:

https://github.com/ampproject/amp.dev/blob/2e3a1fb39f46110065c01ae0f8c8e4a8e11f1965/platform/lib/templates/index.js#L44

Not too sure about the backend but you also get the format the user is on, which might help putting the answers into context:

https://github.com/ampproject/amp.dev/blob/2e3a1fb39f46110065c01ae0f8c8e4a8e11f1965/platform/lib/templates/index.js#L54